sormas-ui/src/main/java/de/symeda/sormas/ui/events/EventDataForm.java

```diff
@@ -372,10 +374,21 @@ protected void addFields() {
 		FieldHelper
 			.setVisibleWhen(getFieldGroup(), EventDto.TYPE_OF_PLACE_TEXT, EventDto.TYPE_OF_PLACE, Collections.singletonList(TypeOfPlace.OTHER), true);
 		setTypeOfPlaceTextRequirement();
-		locationForm.setFieldsRequirement(true, LocationDto.REGION, LocationDto.DISTRICT);
 		locationForm.setFacilityFieldsVisible(getField(EventDto.TYPE_OF_PLACE).getValue() == TypeOfPlace.FACILITY, true);
 		typeOfPlace.addValueChangeListener(e -> locationForm.setFacilityFieldsVisible(e.getProperty().getValue() == TypeOfPlace.FACILITY, true));
 
+		countryField.addValueChangeListener(e -> {
+			CountryReferenceDto serverCountryDto = FacadeProvider.getCountryFacade().getServerCountry();
+			CountryReferenceDto countryDto = (CountryReferenceDto) countryField.getValue();
+			if (countryDto != null && serverCountryDto != null && serverCountryDto.getIsoCode().equalsIgnoreCase(countryDto.getIsoCode())) {
```

Comment:
ISO code part here needs to be changed as well; also, see above (concerning requirements when country == null)

Reply:
ISO part does not need to be changed. _getServerCountry_ returns a _CountryReferenceDto_ based on the country name read from config. Also, from the requirements, in the case of events `region and district are compulsory fields when the server country is selected`.

Reply:
Oh, true, sorry! I misread that. Concerning the second point, that's indeed true, but they also need to be compulsory when no country is selected